### ISSUE TITLE
java.lang.Exception: React Native needs Node.js >= 18

### ISSUE THREAD

**Issue by hadnet** [facebook/react-native#45367]
### Description

I'm trying to open my React Native project in Android Studio, but I'm getting this error shown in the screenshot. When I run 'node -v' in the terminal, it says I have version 20, so I'm not getting why I can't run my RN project.

### Steps to reproduce

1. npx @react-native-community/cli init my-project
2. open RN project in Android Studio

### React Native Version

0.74.3

### Affected Platforms

Runtime - Android

### Output of `npx react-native info`

```text
-
```

### Stacktrace or Logs

```text
Caused by: java.lang.Exception: React Native needs Node.js >= 18. You're currently on version v14.18.1.
```

### Reproducer

https://github.com/facebook/react-native.git

### Screenshots and Videos

![Screenshot 2024-07-11 at 12 03 08 AM](https://github.com/facebook/react-native/assets/13828833/5c29fb13-69fa-408a-b8f4-f4ab97bc33e3)

**Comment by netmaxt3r** [comment]
probably a path issue check with `echo $PATH`  verify `command -v node`
or try using any node managers like [nvm](https://github.com/nvm-sh/nvm)

**Comment by cortinico** [comment]
> probably a path issue check with `echo $PATH` verify `command -v node` or try using any node managers like [nvm](https://github.com/nvm-sh/nvm)

This is the correct answer. Is definitely an environment configuration issue of yours.

**Comment by hadnet** [comment]
> > probably a path issue check with `echo $PATH` verify `command -v node` or try using any node managers like [nvm](https://github.com/nvm-sh/nvm)
> 
> 
> 
> This is the correct answer. Is definitely an environment configuration issue of yours.

Exactly. A NVM path recognition problem. Installing Node.js from .dmg solve the issue. Thanks.

**Comment by MikhilMC** [comment]
> probably a path issue check with `echo $PATH` verify `command -v node` or try using any node managers like [nvm](https://github.com/nvm-sh/nvm)

I am also facing this issue. I am having Ubuntu 22.04 and using node using nvm.

I am getting output for `echo $PATH` as this.
`/home/hp/.nvm/versions/node/v20.10.0/bin:/home/hp/.dotnet/tools:/home/hp/bin:/home/hp/.cargo/bin:/home/hp/.cargo/bin:/home/hp/.local/bin:/home/hp/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/hp/.local/bin/aptos:/home/hp/.foundry/bin:/usr/lib/jvm/java-18-openjdk-amd64/bin:/home/hp/Android/Sdk/tools:/home/hp/Android/Sdk/emulator:/home/hp/Android/Sdk/platform-tools`

And for `command -v node`, I am getting this error.
`/home/hp/.nvm/versions/node/v20.10.0/bin/node`.

Please help me with this.

**Comment by abhilashbhosle** [comment]
Installing the correct node version 18 using nvm and

- rm -rf node_modules package-lock.json
- npm install
- node ./node_modules/react-native/cli.js start --reset-cache

worked for me.
